pkg/metrics/metrics.jsx

```diff
@@ -238,6 +238,18 @@ function make_rows(rows, rowProps, columnLabels) {
     );
 }
 
+async function get_pcp_packages() {
```

Comment:
Commit message: Please say "pcp" (or both), as it's really not just about the Python binding.